lib/templates/android_screen_base.tt

```diff
@@ -16,37 +16,37 @@ class AndroidScreenBase < Calabash::ABase
     start_test_server_in_background
   end
 
-  def method_missing(method, *args)
-    if method.to_s.start_with?('touch_')
-      # If method name starts with touch_, executes the touch
-      # screen element method using the element name which is the
-      # method name without the first 'touch_' chars
-      touch_screen_element public_send(method.to_s.sub('touch_', ''))
-    elsif method.to_s.start_with?('enter_')
-      # If method starts with enter_, execute the enter method using
-      # the field name, which is the method name without the initial
-      # 'enter_' chars and appended '_field' chars
+  def <%= I18n.translate( "screen_base.method_missing" ) %> (method, *args)
+    if method.to_s.<%= I18n.translate "screen_base.start_with?" %>('<%= I18n.translate "screen_base.touch_" %>')
+      # <%= I18n.translate( "comments.method_missing_1" ) %>
+      # <%= I18n.translate( "comments.method_missing_2" ) %>
```

Comment:
This separation in many translation keys is no good. What if we wanna change these comments? If we wanna remove one line or add another?

The best solution here is to create only one translation key and create a helper method to "print" comments. In Ruby, the standard is that lines should not have more than 80 chars. So you could create a method that break the line every time you reach 80 chars. Remember to not split words, you should break the line in the last space char before you reach 80 chars.